Source/inv.h

```diff
@@ -19,6 +19,7 @@ namespace devilution {
 #define INV_SLOT_HALF_SIZE_PX (INV_SLOT_SIZE_PX / 2)
 #define INV_ROW_SLOT_SIZE 10
 constexpr Size InventorySlotSizeInPixels { INV_SLOT_SIZE_PX };
+static constexpr uint16_t InvalidItemId = uint16_t(-1);

```

Comment:
This constant is used in comparisons with an index into the global Stash's internal item list. Should probably be defined in stash.h instead of inv.h. It's used more like a nullptr value (or end iterator/`npos` style invalid position) so a name that doesn't reference "ID" would be nice.

Reply:
Suggestion applied in most recent push (removed all InvalidItemId references and reverted to `-1`)